### PR TITLE
Reduce duplicate code by reusing filter panel

### PR DESCRIPTION
**Pull request by signedav**
wizard page for export uses the same panel to filter the data as the validator

### CODE REVIEW

**Comment by signedav** [comment]
Might conflict #614

File: QgisModelBaker/gui/workflow_wizard/workflow_wizard.py [hunk 1]
```diff
@@ -336,15 +336,17 @@ def id_changed(self, new_id):
...
-                models = self.export_models_model.checked_entries()
-            elif self.current_export_filter == SchemaDataFilterMode.DATASET:
-                datasets = self.export_datasets_model.checked_entries()
-            elif self.current_export_filter == SchemaDataFilterMode.BASKET:
-                baskets = self.export_baskets_model.checked_entries()
+            if self.current_filter_mode == SchemaDataFilterMode.MODEL:
+                models = self.current_models_model.checked_entries()
+            elif self.current_filter_mode == SchemaDataFilterMode.DATASET:
+                datasets = self.current_datasets_model.checked_entries()
+                print(datasets)
```
Comment (domi4484) [inline QgisModelBaker/gui/workflow_wizard/workflow_wizard.py:343]: It's a forgotten debug print or it's intended to stay?

File: QgisModelBaker/gui/workflow_wizard/workflow_wizard.py [hunk 2]
```diff
@@ -336,15 +336,17 @@ def id_changed(self, new_id):
...
+            elif self.current_filter_mode == SchemaDataFilterMode.BASKET:
+                baskets = self.current_baskets_model.checked_entries()
+                print(baskets)
```
Comment (domi4484) [inline QgisModelBaker/gui/workflow_wizard/workflow_wizard.py:346]: same here...

**Review by domi4484 — APPROVED** [review]

File: QgisModelBaker/gui/workflow_wizard/workflow_wizard.py (hunk 1 above)
Comment (signedav) [reply QgisModelBaker/gui/workflow_wizard/workflow_wizard.py:343]: Debug print... Thanks for spotting.

Comment (signedav) [reply QgisModelBaker/gui/workflow_wizard/workflow_wizard.py:343]:
```suggestion
```

File: QgisModelBaker/gui/workflow_wizard/workflow_wizard.py (hunk 2 above)
Comment (signedav) [reply QgisModelBaker/gui/workflow_wizard/workflow_wizard.py:345]:
```suggestion
```

**Pull request merged**